e2e/scaffolding/helpers.ts

```diff
@@ -93,6 +97,50 @@ export function signPayload(keys: KeyringPair, data: Codec): MultiSignatureType
   }
 }
 
+/**
+ * Signing EIP-712 compatible signature for AddKeyData
+ * @param keys
+ * @param payload
+ */
+export async function signEip712AddKeyData(keys: Keypair, payload: AddKeyData): Promise<MultiSignatureType> {
```

Comment:
a temporary function until we implement the common library for Ethereum utilities